examples/fit_fake_data.py

```diff
@@ -21,7 +21,6 @@
 tau = nddata(r_[0:2:100j], 'tau')
 fake_data = 102*(1-2*exp(-tau*6.0))
 fake_data.add_noise(5.0)
-empty_data = nddata(r_[0:2:100j],'tau')
```

Comment:
shouldn't be needed -- wasn't part of the code before, right?

Reply:
I need this to initiate (instantiate?) the class with empty data before fitting the data into it. This is also how it's done in the fit with bounds example. Without it it does not fit appropriately.